examples/AksCluster/AKSCluster.Tests.ps1

```diff
@@ -2,14 +2,52 @@ BeforeAll {
   Import-Module Az.InfrastructureTesting
 }
 
+$resourceType = "AksCluster"
+$resourceName = "aksbenchpresstest"
+$rgName = "rg-test"
+
+Describe 'Verify AKS Cluster with Confirm-AzBPResource' {
+  it 'Should contain an AKS Cluster named aksbenchpresstest' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  it 'Should contain an AKS Cluster named aksbenchpresstest with agent pool named agentpool' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+      PropertyKey       = "AgentPoolProfiles[0].Name"
```

Comment:
Super happy to see the use of an array index in the key!

Reply:
Yeah glad it works!